main.go

```diff
@@ -307,11 +310,11 @@ func printFileAnalysis(file string, limit int64, isJSON bool) error {
 
 	fmt.Printf(
 		`%s: %d lines (%d sloc)
-  type:      %s
-  mime_type: %s
-  language:  %s
-  vendored:  %t
-`,
+				type:      %s
```

Comment:
this should not be changed as it changes output

Reply:
Undid the changes.